libs/shared/ui-components/src/Team/TeamMember/TeamMemberName.tsx

```diff
@@ -5,7 +5,7 @@ import clsx from 'clsx';
 import { TTeamMemberName } from './types';
 
 export const TeamMemberName: FC<TTeamMemberName> = ({ name }) => (
-  <h3 className="relative mb-[1rem] h-[5rem] md:h-[6rem]">
+  <div className="relative mb-[1rem] h-[5rem] md:h-[6rem]">
```

Comment:
I think it's heading overload to give each team member on the page a separate heading. The idea of headings is to help someone navigate to major sections of content. The list of team members IS the section of content and it doesn't have any internal sectioning. That's my logic, but I could be wrong.